python/ray/train/accelerators/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .base import Accelerator  # noqa: F401
+
+try:
+    import torch
+    from .torch import TorchAccelerator  # noqa: F401
+except ImportError:
+    pass
```

Comment:
Motivation for this approach is two-fold:

One, so that we can do
```
from ray.train.accelerators import Accelerator, TorchAccelerator
```
instead of 
```
from ray.train.accelerators.base import Accelerator
from ray.train.accelerators.torch import TorchAccelerator
```

Two, because this approach abstracts the file structure of the `accelerators` package. We can freely add, remove, or rename modules in the `accelerators` package without breaking modules that import from the `accelerators` package.